travis/run_tests.sh

```diff
@@ -208,28 +208,28 @@ done
 if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then bash <(curl -s https://codecov.io/bash) -F tests ; fi
 
 # After all tests are run, exit with a FAIL if $fail_counter>0, otherwise PASS.
-if [[ "$style_test_passed" -gt 0 ]]; then
```

Comment:
Are these lines meant to be commented out?

Reply:
Sorry, just seen your edited comment about the tests. I think this is here to provide a summary of all the tests in one place at the end of the testsuite - so if something fails, you can see it here. I found it made the Travis logs a little easier to navigate. It's up to you whether you want to remove it or not.

Reply:
Yes I see that, but `make tests` does not execute the indent and style tests so this always report them as passed which is pointless. It makes sense only for `make alltests` in which case I agree it is useful. Unless I am reading this wrong.

Reply:
You're absolutely right.

Reply:
I will remove these lines then. But I agree with you that a summary should be printed at the end of the test suite. I will create a new issue about that.